apps/src/code-studio/components/AssetManager.jsx

```diff
@@ -139,11 +147,24 @@ export default class AssetManager extends React.Component {
     });
   };
 
+  afterAudioSaved = (err) => {
+    this.setState({recordingAudio: false});
+    this.setState({audioErrorType: err});
+  };
```

Comment:
I think you can combine these lines to: `this.setState({
recordingAudio: false, 
audioErrorType: err,
})`